wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/StringEscapeHTMLToXML.java

```diff
@@ -0,0 +1,39 @@
+package com.github.bordertech.wcomponents.util;
```

Comment:
I am not sure what you mean here. By extending StringEscapeUtils we can use this class to also call the static final methods in StringEscapeUtils (such as escapeXML10). It is also a different thing from a sanitizer: this does not sanitize, it unescapes: for example `&bul;` to `•`